offchain/test-fixtures/src/broker_redis.rs

```diff
@@ -0,0 +1,24 @@
+use testcontainers::{
```

Comment:
This file is particular to the rollups-events lib and won't be used elsewhere. Let's go ahead and move it there.